tests/OK/inductive_type.lp

```diff
@@ -0,0 +1,11 @@
+inductive nat : TYPE ≔
+ | z    : nat
+ | succ : nat → nat
+
+inductive list ≔
+ | nil  : list
+ | cons : a → list → list
+
+symbol nat : TYPE
```

Comment:
If `inductive nat` generates a symbol `nat`, this command will fail as one cannot define the same object twice in the same file.